pkg/capabilities/cli/cmd/testdata/streams.capability.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
Will move it to an actual streams folder in a follow-up. This should be the real deal (today; duplicating for testing purposes as it might change later).